pkg/server/server.go

```diff
@@ -0,0 +1,37 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/harrisonlob/goyagi/pkg/health"
+	"github.com/labstack/echo"
+	"github.com/lob/logger-go"
+	"github.com/harrisonlob/goyagi/pkg/signals"
+)
+
+// New returns a new HTTP server with the registered routes.
+func New() *http.Server {
+	log := logger.New()
+
+	e := echo.New()
+
+	health.RegisterRoutes(e)
+	srv:= &http.Server{
```

Comment:
Let's add a space before the `:=` here too!

Reply:
@virtualdom fixed!